backend/rates/history.go

```diff
@@ -49,16 +49,43 @@ var (
 	}
 )
 
+// EnableHistoryPair spins up a new goroutine to periodically update
+// and backfill historical exchange rates.
+// If a pair is unsupported or already enabled, EnableHistoryPair does nothing.
+// To stop periodic updates for this coin/fiat pair, the callers can use DisableHistoryPair.
+func (updater *RateUpdater) EnableHistoryPair(coin, fiat string) {
+	if geckoCoin[coin] == "" || geckoFiat[fiat] == "" {
+		return
+	}
+	updater.historyMu.Lock()
+	defer updater.historyMu.Unlock()
+	ctx, cancel := context.WithCancel(context.Background())
+	go updater.historyUpdateLoop(ctx, coin, fiat)
+	go updater.backfillHistory(ctx, coin, fiat)
+	updater.historyGo[coin+fiat] = cancel
+}
+
+// DisableHistoryPair deactivates backfill and periodic updates
+// of historical exchange rates for the given coin/fiat pair.
+// If a pair is unsupported or wasn't active, DisableHistoryPair does nothing.
+func (updater *RateUpdater) DisableHistoryPair(coin, fiat string) {
```

Comment:
This is never called. It should probably be called in the place where you call `EnableHistoryPair` in the backend, for all other pairs that are not active?

Maybe easiest to by default disable all possible pairs at start, then enable the active pairs after.

Reply:
True, I added it for consistency, to be used in the next commit where pairs are enabled/disabled at runtime from the UI, but I have another idea how to disable them.

Removed because all pairs are in fact disabled by default at the startup.